rasa/utils/tensorflow/models.py

```diff
@@ -69,7 +67,223 @@
 
 
 # noinspection PyMethodOverriding
-class RasaModel(tf.keras.models.Model):
+class TmpKerasModel(tf.keras.models.Model):
```

Comment:
let's put it in a separate file to complete the separation. I think we can put it to the same file as data_adapter and maybe name it temp_keras.py or smth